src/js/features/escrow/saga.js

```diff
@@ -28,6 +28,9 @@ import {ADD_OFFER_SUCCEEDED} from "../metadata/constants";
 
 const { toBN } = web3.utils;
 
+window.MetadataStore = MetadataStore;
```

Comment:
This is for the trick of creating escrows without requiring using metamask, right?

Reply:
Oops, yes, my bad.